modules/core/src/main/scala-3/dev.profunktor.redis4cats/TypeInqualityCompat.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018-2021 ProfunKtor
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.profunktor.redis4cats
+
+import scala.util.NotGiven
```

Comment:
Nitpicking: we could probably have implemented a NotGiven polyfill for 2.13 and define `=!=` using the same code, but it's probably simpler as it is.